kdl_parser/src/kdl_parser.cpp

```diff
@@ -86,9 +88,10 @@ KDL::Joint toKdl(urdf::JointSharedPtr jnt)
         return KDL::Joint(jnt->name, F_parent_jnt.p, F_parent_jnt.M * axis, KDL::Joint::TransAxis);
       }
     default: {
-        fprintf(
-          stderr, "Converting unknown joint type of joint '%s' into a fixed joint\n",
-          jnt->name.c_str());
+        rcutils_log(nullptr,
```

Comment:
Can we instead use:

`RCUTILS_LOG_WARN_NAMED` here?

(same goes for all of the other uses below)